cloud/google/machineactuator.go

```diff
@@ -103,6 +112,15 @@ func NewMachineActuator(kubeadmToken string, machineClient client.MachineInterfa
 		}
 	}
 
+	// TODO: get rid of empty string check when we switch to the new bootstrapping method.
+	var configWatch *machinesetup.ConfigWatch
+	if configListPath != "" {
+		configWatch, err = machinesetup.NewConfigWatch(configListPath)
+		if err != nil {
+			glog.Errorf("Error creating config watch: %v", err)
```

Comment:
Was it deliberate that this is swallowing the error instead of returning it? If so, I think it's worth a comment why, because that's usually an anti-pattern and could use justification.